docs/concepts/copilot/copilot.md

```diff
@@ -0,0 +1,80 @@
+
+## Knowledge Graph
+
+Data Copilot works by enhancing the user’s prompt using a knowledge graph.
+
+![Knowledge Graph](img/copilot_knowledge_graph.png)
+
+Our knowledge graph includes metadata for a Project’s entities (such as Datasets, Schemas, Seeds, Models, and Pipelines) and the statistical usages of these entities. We built the knowledge graph intentionally to include metadata but not data. The Dataset structure is included in the knowledge graph but individual Dataset records are not.
+
+![Architecture](img/copilot_arch.png)
+
+Prophecy sends the enhanced Prompt to OpenAI. The model returns SQL or Spark code, and Prophecy verifies whether this code is valid. Once Prophecy fixes the code, Prophecy generates a visual Pipeline and displays this Pipeline to you for review.
+
+## Deployment
+
+Data Copilot is available for all customers using Prophecy’s managed Public SaaS offering and uses the public SaaS version of OpenAI's Language model. Customers using the Private SaaS or on-prem offerings can enable Data Copilot as a flag in the deployment / upgrade configuration.
+
+For more details, see [Enable Data Copilot](/concepts/copilot/enable-data-copilot).
+
+## FAQ
+
+###
+
+#### How is Prophecy Copilot different than Github Copilot?
+
+Github Copilot is a great tool to boost productivity and extend the reach of the coding community. However, not every problem is solved with direct coding. More importantly, users need a Copilot with some context of the interesting data.
+
+For teams of analysts, data platform providers, or line-of-business users, the pain points are not resolved by teaching every team member how to code. Data Copilot empowers less technical users because they don’t have to code. Importantly, technical and coding users benefit from Data Copilot because visual Pipelines are easier to understand, explain, and leverage. Prophecy’s Data Copilot boosts the productivity of the business user and the more technical coding team members.
+
+When all of these data practitioners reach for an AI assistant, they’ll need one specific to the data space. For example, the assistant should collect metadata from the Databricks Workspace. Prophecy Data Copilot has the context of your data, and it can guide Pipeline and model development by suggesting which Datasets to use and how to use them.
+
+Github Copilot and Prophecy Data Copilot are both excellent tools to boost productivity, but Prophecy Data Copilot is accessible to a larger user base and can make data suggestions because it maintains data context.
+
+#### Which Datasets are accessible to Prophecy Data Copilot?
+
+Any Dataset, Source, or Seed that is part of the linked (Databricks, Snowflake, etc.) catalog is accessible. That is, if you can access the Datasets with your personal access token, then the Datasets should appear in the Environment tab and Copilot can access them.
```

Comment:
I suggest adding: 

Add any of these datasets from the Environment tab into your Project canvas. Once the dataset is defined in the project, Copilot's AI capabilities will start making recommendations including that data source.

Reply:
Done.